### DISCUSSION TITLE
What is the theme that used in this template?

### DISCUSSION THREAD

**Discussion by yuxxeun** [arthelokyo/astrowind#50] [Q&A]
where to change tag `a` color in markdown? i was add `!important` in global css file, but doesn't work

**Comment by prototypa** [comment]
Hi @yuxxeun 

For typography in markdown we use https://tailwindcss.com/docs/typography-plugin

> "The official Tailwind CSS Typography plugin provides a set of prose classes you can use to add beautiful typographic defaults to any vanilla HTML you don’t control, like HTML rendered from Markdown, or pulled from a CMS."

In case of links you can do something like this:

```html
<article class="prose prose-a:text-blue-600">
  {{ markdown }}
</article>
```

More information in https://tailwindcss.com/docs/typography-plugin#element-modifiers

**Reply by yuxxeun** [reply]
yes i was changed it yesterday, i searched and finally found it, thanks anyway
